pkg/manager/client/discovery.go

```diff
@@ -31,40 +30,35 @@ func NewDiscoveryClient(ctx context.Context, config *rest.Config) (*DiscoveryCli
 	}, nil
 }
 
-func toObj(b []byte, groupVersion string, kind string) interface{} {
-	re, err := regexp.Compile(`""`)
-	if err != nil {
-		logrus.Errorf("Unable to create regex.")
-		return nil
-	}
+func toObj(b []byte, namespace, groupVersion, kind string) interface{} {
 
-	replacement := `"null"`
-	afterReplace := re.ReplaceAllLiteralString(string(b), replacement)
+	replaceString := strings.ReplaceAll(string(b), `"creationTimestamp":null`, `"creationTimestamp":"null"`)
+	replaceString = strings.ReplaceAll(replaceString, `\"creationTimestamp\":null`, `\"creationTimestamp\":\"null\"`)
 
-	jsonParsed, err := gabs.ParseJSON([]byte(afterReplace))
+	finalString := strings.ReplaceAll(replaceString, `""`, `"null"`)
+	jsonParsed, err := gabs.ParseJSON([]byte(finalString))
 	if err != nil {
-		logrus.Errorf("Unable to parse json.")
+		logrus.Errorf("Unable to parse json: %s, %s, %s", namespace, groupVersion, kind)
 		return nil
 	}
-
 	// the yaml contains a list of resources
-	_, err = jsonParsed.SetP("List", "kind")
-	_, err = jsonParsed.SetP("apps/v1", "apiVersion")
+	if _, err = jsonParsed.SetP("List", "kind"); err != nil {
```

Comment:
It seems this commit has similar functionality as the first one. We should merge them if it's the case.